src/StreamJsonRpc/JsonMessageFormatter.cs

```diff
@@ -40,7 +45,7 @@ public class JsonMessageFormatter : IJsonRpcMessageTextFormatter
         /// that uses <see cref="Encoding.UTF8"/> for its text encoding.
```

Comment:
If the default is different, then this comment should probably be updated. I don't `DefaultEncoding` value will be visible to people other than those who read the code.

Reply:
Updated. :)